tests/drivers/pwm/pwm_gpio_loopback/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Test requires wire connection between:
+ *  - PWM130 OUT[0] at P0.00 <-> GPIO input at P0.01
+ *  - PWM120 OUT[0] at P7.00 <-> GPIO input at P1.09
+ *  - PWM120 OUT[1] at P7.01 <-> GPIO input at P1.05
+ */
+
+#include <zephyr/dt-bindings/pwm/pwm.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/gpio/nordic-nrf-gpio.h>
```

Comment:
Definitions from this file do not seem to be used here. But anyway, it would be better to add this inclusion to `nrf_common.dtsi` to avoid repeating it.